Tasks/AzureAppServiceManageV0/task.json

```diff
@@ -187,6 +187,15 @@
             },
             "visibleRule": "Action == Enable Continuous Monitoring",
             "helpMarkDown": "Select Application Insights resource where continuous monitoring data will be recorded. <br/>If your application insights resource is not listed here and you want to create a new resource, click on [+New] button. Once the resource is created on Azure Portal, come back here and click on refresh button."
+        },
+        {
```

Comment:
Move this input under 'Advanced settings'

Reply:
@hkamel  can you give me collaborator permission so that I can update the required changes to your PR.